h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -21,10 +30,16 @@
   private static int treeToPrint = -1;
   private static int maxLevelsToPrintPerEdge = 10;
   private static boolean detail = false;
-  private static String outputFileName = null;
+  private String outputFileName = null;
   private static String optionalTitle = null;
   private static PrintTreeOptions pTreeOptions;
   private static boolean internal;
+  private String outputFormat = "dot";
```

Comment:
@michalkurka  maybe we should first merge https://github.com/h2oai/h2o-3/pull/4104 and build this on top of it?